src/CONST.js

```diff
@@ -476,7 +476,7 @@ const CONST = {
             SPLIT: 'split',
             REQUEST: 'request',
         },
-        AMOUNT_MAX_LENGTH: 10,
+        AMOUNT_INTEGER_PART_MAX_LEN: 7,
```

Comment:
I commented on a out of date thread, so leaving it here too: 

Can you reset the value to 10 please, we specifically want to support very large IOU amounts.

Reply:
We are still supporting the same range of values. The purpose of this constant has been changed now as per the validation conditions. This represents the integer part length of the amount now. So we have  (10 -3) = 7 as the value. We were supporting 10 digit strings before and still are! Now we have `AMOUNT_INTEGER_PART_MAX_LEN=7` digits before the decimal point And `2` after the decimal point. So we still support the max length of string = 10.

Reply:
Ah I didn't read it properly... That makes sense, thanks for explaining.